tests/unit/cube/qb/test_qb_unit.py

```diff
@@ -42,6 +50,91 @@ def test_new_unit_base_unit_validation():
     )
 
 
+def test_scaling_factor_defined():
+    with TemporaryDirectory() as t:
+        temp_dir = Path(t)
+
+        cube_config = {
+            "columns": {
+                "Amount": {
+                    "type": "observations",
+                    "unit": {
+                        "label": "Count",
+                        "from_existing": "http://qudt.org/vocab/unit/NUM",
+                        "scaling_factor": 0.1,
+                    },
+                }
+            }
+        }
+
+        data = pd.DataFrame({"Dimension 1": ["A", "B", "C"], "Amount": [100, 200, 300]})
+
+        data_file_path = temp_dir / "data.csv"
+        config_file_path = temp_dir / "config.json"
+
+        with open(config_file_path, "w+") as config_file:
+            json.dump(cube_config, config_file, indent=4)
+
+        data.to_csv(str(data_file_path), index=False)
+
+        deserialiser = get_deserialiser(SCHEMA_PATH_FILE, 3)
+
+        cube, _, _ = deserialiser(data_file_path, config_file_path)
+        amount_col = cube.columns[1]
+
+        assert_num_validation_errors(amount_col.pydantic_validation(), 0)
+
+        assert isinstance(amount_col, QbColumn)
+        assert isinstance(
+            amount_col.structural_definition, QbMultiMeasureObservationValue
+        )
+        unit = amount_col.structural_definition.unit
+        assert isinstance(unit, NewQbUnit)
+        assert unit.base_unit_scaling_factor == 0.1
+
+
+def test_scaling_factor_not_defined():
+    with TemporaryDirectory() as t:
+        temp_dir = Path(t)
+
+        cube_config = {
+            "columns": {
+                "Amount": {
+                    "type": "observations",
+                    "unit": {
+                        "label": "Count",
+                        "from_existing": "http://qudt.org/vocab/unit/NUM",
```

Comment:
We could probably use a third test to ensure that if you specify a new unit without basing it off an existing one then you end up with no scaling factor defined.

At that point you may want to refactor the unit test into a function like the following:

```python
unit = _test_unit_definition({ 
  "label": "Count",
  "from_existing": "..."
})
assert isinstance(unit, NewQbUnit)
assert unit.base_unit_scaling_factor ....
```

edit: I've just seen the following file which pretty much covers that base - there's a fair bit of overlap between the two set of tests.